### ISSUE TITLE
Installation failure of background music pkg file 

### ISSUE THREAD

**Issue by youjust19** [kyleneideck/BackgroundMusic#389]
<img width="659" alt="Screenshot 2020-11-17 at 9 53 13 PM" src="https://user-images.githubusercontent.com/53900885/99416870-4d261b00-291f-11eb-89a9-f4dca5a3c8a1.png">

Hi Team, 

I am trying to instal the latest pkg file, but it is failing at installation. I have recently upgrade MAC to Big Sur

**Comment by sammy183** [comment]
I don't believe Background Music is working with Big Sur as of now. I had to uninstall it after updating because it just muted everything and didn't work.
